src/app/routes/resetpwd/resetpwd.component.html

```diff
@@ -97,8 +97,106 @@ <h1>You have requested to reset your password</h1>
             </td>
         </tr>
     </table>
+    </div>
     <!--/100% body table-->
+    <div class="d-block d-md-none rwdcontainer reset">
+      <table
+      style="@import url(https://fonts.googleapis.com/css?family=Rubik:300,400,500,700|Open+Sans:300,400,600,700); font-family: 'Open Sans', sans-serif;">
```

Comment:
again, create class to apply this, importing from url should be done in styles or index file